### ISSUE TITLE
pCloud sync doesn't work

### ISSUE THREAD

**Issue by babb0** [Kunzisoft/KeePassDX#481]
The file.kdbx was created by KeePassXC and stored on pcloud-drive to sync with my phone (worked fine with Keepass2Andriod). After trying to open the file with KeePassDX (fresh installed) remained an empty file.kdbx on my pcloud drive. 
A test with a new file test.kdbx with only one entry brought the same result.

** Keepass Database **
Created with: Ubuntu 1910 snap KeePassXC 2.5.3
Location: pCloud sync folder
Size: 1MB
Contains attachment: no

KeePassDX (please complete the following information):
Version: 2.5beta27
Language: german

Android (please complete the following information):
Device: Motorola Moto G6 plus
Version: 9

**Comment by J-Jamet** [comment]
Does it happen locally in your device by not using pCloud?

**Comment by babb0** [comment]
Good morning :-)

i think it wasn't locally, i used the file from the connected/mounted
pcloud drive (i have no offline pcloud files on the phone).
I clicked on the button "ad an existing file", the file-browser opened and
i choose the pcloud-drive within i tried to open the file.kdbx. I added the
key-file but, i do not exactly remember the sequence. I coud not open the
file.kdbx and pcloud synced immediately an empty file.kdbx.

Have a nice day



Am So., 8. März 2020 um 22:38 Uhr schrieb Jérémy JAMET <
notifications@github.com>:
